STUDIO/org.openl.rules.webstudio/webapp/pages/modules/administration/users/groups.xhtml

```diff
@@ -468,8 +468,9 @@
                 RichFaces.$("modalAddEditGroup").hide();
                 location.reload();
             }).fail(function (response) {
-                if (response.responseText) {
-                    ws.ui.error(response.responseText);
+                let json = JSON.parse(response.responseText);
+                if (json.message) {

```

Comment:
What if response.responseText is null in the line above?
I think we should check that the object "json" isn't null, like:
```suggestion
                if (json && json.message) {
```

Also could you check other places where response.responseText is used please? For example, I think we should make same changes (parse json instead of using text as is) in the webapp/pages/modules/index.xhtml

Reply:
Updated